docs/manual/install/index.rst

```diff
@@ -67,14 +67,12 @@ You'll also need ``libpangocairo``, which on Ubuntu can be installed via ``sudo
 apt-get install libpangocairo-1.0-0``. Qtile uses this to provide text
 rendering (and binds directly to it via cffi with a small in-tree binding).
 
-dbus/gobject
+dbus_next
 ------------
 
-Until someone comes along and writes an asyncio-based dbus library, qtile will
-depend on ``python-dbus`` to interact with dbus. This means that if you want
-to use things like notification daemon or mpris widgets, you'll need to
-install python-gobject and python-dbus. Qtile will run fine without these,
-although it will emit a warning that some things won't work.
+Qtile requires ``dbus_next`` which is a lightweight, asyncio-based dbus library
+to interact with dbus and provide things like notification daemon or mpris
+widgets.
```

Comment:
I guess we should just include dbus_next in setup.cfg and remove that part of the doc.

Reply:
I've added dbus_next to setup.cfg but have left this bit in the docs as that's in the "installing from source" section so it's correct to show the dependency.

Reply:
The `pip install .` in the next section will already install it if it's in the setup.cfg.

Reply:
Ah. I understand now. Will fix.